### ISSUE TITLE
How can I custom Android Card Component UI?

### ISSUE THREAD

**Issue by zhangduyu** [Adyen/adyen-android#625]
Hi,
If I can custom Android Card Component UI? 
I didn't find the solution from official document.

I'd like to modify the card component like our UI design:
![image](https://user-images.githubusercontent.com/7981548/160056030-b50524d9-4e5e-4fca-b75c-a63bbda58871.png)

**Comment by caiofaustino** [comment]
Hey there, yeah sorry, the documentation for this is not available yet.
But the general idea is that you can override the themes and styles of the SDK on your app to customize them however you like.
You can fin the most relevant styles [here](https://github.com/Adyen/adyen-android/blob/develop/ui-core/src/main/res/values/styles.xml) and [here](https://github.com/Adyen/adyen-android/blob/develop/card/src/main/res/values/styles.xml).

**Comment by zhangduyu** [comment]
> Hey there, yeah sorry, the documentation for this is not available yet. But the general idea is that you can override the themes and styles of the SDK on your app to customize them however you like. You can fin the most relevant styles [here](https://github.com/Adyen/adyen-android/blob/develop/ui-core/src/main/res/values/styles.xml) and [here](https://github.com/Adyen/adyen-android/blob/develop/card/src/main/res/values/styles.xml).

Hi Caio,
Thanks for you replying, it helped a lot.
I still have some questions.

1 Is it possible I can change the TextInput(Card number/Expire date/CVV) style to default? Currently the style is "OutlinedBox".
![image](https://user-images.githubusercontent.com/7981548/160536135-c39a787e-68f7-46a9-b664-5b66eb146693.png)

2 How can I change the "Save Card" switch's tint color?
![image](https://user-images.githubusercontent.com/7981548/160536473-0cdc5567-bb03-4ebf-aeed-394245fdf144.png)

**Comment by caiofaustino** [comment]
Yeah, you should be able to just override [`AdyenCheckout.TextInputLayout`](https://github.com/Adyen/adyen-android/blob/develop/ui-core/src/main/res/values/styles.xml#L35).
